packages/x-codemod/src/v6.0.0/data-grid/row-selection-props-rename/actual.spec.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import { DataGrid } from '@mui/x-data-grid'
```

Comment:
Nitpick: You could also consider adding a case for `DataGridPremium`. 🙈 
```suggestion
import * as React from 'react';
import { DataGrid } from '@mui/x-data-grid';
import { DataGridPro } from '@mui/x-data-grid-pro';
```